usmqe/ceph/commands.py

```diff
@@ -10,6 +13,11 @@
 
 
 LOGGER = pytest.get_logger('usmceph.commands', module=True)
+usmqe.usmssh.KEYFILE = "~/.ssh/id_rsa"
+try:
+    usmqe.usmssh.KEYFILE = pytest.config.getini("usm_ssh_keyfile")
+except ValueError:
+    pass
```

Comment:
@mkudlej try to update usmssh.py similarly to this patch: https://github.com/dahorak/usmqe-tests/commit/2622e98224d1d13dd2ccad6cda0efa26107ba401
I think it should work, if not, we will have to find the root cause and fix it. I've tried it on similar testing code and it works as expected.